### PR TITLE
make some small improvements

### PR DESCRIPTION
**Pull request by Emeryao**

### CODE REVIEW

File: selectlinestatusbar.js [hunk 1]
```diff
@@ -7,14 +7,24 @@ const
...
+    }
+    this._statusBar = vscode.window.createStatusBarItem(alignment, vscode.workspace.getConfiguration('selectline').statusbarPriority || 100);
+    vscode.window.onDidChangeActiveTextEditor(e => e && this.displaySelectedLineCount(e.selections));
+    vscode.window.onDidChangeTextEditorSelection(e => e && this.displaySelectedLineCount(e.selections));
+    vscode.window.onDidChangeTextEditorViewColumn(e => e && this.displaySelectedLineCount(e.selections));
     this._statusBar.show();
   }
   displaySelectedLineCount(selections) {
-    let selectedcount = selections.reduce((pre, selection) => pre + selection.end.line - selection.start.line + 1, 0);
+    console.log(selections);
```
Comment (tomoki1207) [inline selectlinestatusbar.js]: Please remove debug codes.

File: selectlinestatusbar.js [hunk 2]
```diff
@@ -7,14 +7,24 @@ const
 
 class SelectLineStatusBar {
   constructor() {
-    this._statusBar = vscode.window.createStatusBarItem();
-    vscode.window.onDidChangeActiveTextEditor(e => this.displaySelectedLineCount(e.selections));
-    vscode.window.onDidChangeTextEditorSelection(e => this.displaySelectedLineCount(e.selections));
-    vscode.window.onDidChangeTextEditorViewColumn(e => this.displaySelectedLineCount(e.selections));
+    let alignment = vscode.StatusBarAlignment.Left;
+    let alignConfig = vscode.workspace.getConfiguration('selectline').alignment;
```
Comment (tomoki1207) [inline selectlinestatusbar.js]: It seems that we should restart VSCode for these settings to be reflected.
I think it is better to configure these while checking the change result every save the `settings.json`.

**Comment by tomoki1207** [comment]
@Emeryao Thank you for your work!
Could you fix these points and rebase your commits?

**Comment by Emeryao** [comment]
Hi @tomoki1207 
I've fixed your reviews and make another PR with a branch that rebased all my commits

**Pull request closed without merging**